test/apps/test_init_mt.c

```diff
@@ -14,10 +14,21 @@
 #include <omp.h>
 #endif
 
+#include <time.h>
+#include <sys/time.h>
+#include <stdlib.h>
```

Comment:
can some of these be removed?

Reply:
> can some of these be removed?

sys/time.h - gettimeofday
sodlib.h - EXIT_SUCCESS - removed